src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  display: flexbox;
+}
+
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: dotted 1vmin #000;
+  border-radius: 50%;
+}
+
+.stopwatch::before {
+  z-index: 1;
+  content: '';
+  position: absolute;
+  bottom: 40vmin;
+  left: 38.5vmin;
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  animation: speed-up 600s steps(10) alternate;
+  transform-origin: bottom left;
+}
+
+.stopwatch::after {
+  z-index: 2;
+  content: '';
+  position: absolute;
+  bottom: 40.25vmin;
+  left: 39.5vmin;
+
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  animation: speed-up 10s linear infinite;
+  transform-origin: bottom left;
+}
+
+span {
```

Comment:
better use class

Reply:
I can't understand why  DEMO on github is empty.